src/shogun/labels/Labels.h

```diff
@@ -96,6 +96,12 @@ class CLabels : public CSGObject
 	 * Calls subset_changed_post() afterwards */
 	virtual void remove_all_subsets();
 
+	/** returns subset stack
```

Comment:
Question: Does it need to be virtual? I know the other one is, but are there cases where we want this? And ... const?